src/mixins/placeholder.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+/**
+ * CSS to style the selection psuedo-element.
+ *
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   ...placeholder(styles)
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.input`
+ *   ...placeholder(styles)
```

Comment:
Would `placeholder` need its types amended though? It's` css` that is modifying `placeholder`'s output to an array. If another JS to CSS library implemented another means that changed the type we would need to add that too.